lib/util/eventobserver.js

```diff
@@ -56,15 +57,16 @@ class EventObserver extends EventEmitter {
         const publisherPayload = Object.assign(payload ? payload : {}, { elapsedTime, level });
         this._publisher.publish(group, name, publisherPayload);
       }
+      const event = Object.assign(payload ? { payload } : {}, {
+        elapsedTime,
+        group,
+        level,
```

Comment:
Should we remove `level`? This is already available in the original logger method factory. It'd be good to advise users to use that when determining log level for any type of log.

```js
var originalFactory = logger.methodFactory;
logger.methodFactory = function (methodName, level, loggerName) {
  var method = originalFactory(methodName, level, loggerName);
  return function (datetime, logLevel, component, message, data) {
    method(datetime, logLevel, component, message, data);

    console.log(level); // prints debug, info, warn, or error
    console.log(logLevel); // prints 0 - 3. These are coming from Log.DEBUG, Log.INFO, etc
  };
};
```

Reply:
event level tells the severity of the event ( for example signaling `closed` would fire at level='error'. I am taking your suggestion to use log those at appropriate logLevel. But I think we do need to keep these.